app/src/test/java/de/westnordost/streetcomplete/quests/opening_hours_signed/CheckOpeningHoursSignedTest.kt

```diff
@@ -67,6 +69,16 @@ class CheckOpeningHoursSignedTest {
             "opening_hours:signed" to "yes"
         ))))
     }
+    @Test fun `is applicable if the opening hours not signed and only check date for OH exists`() {
+        assertTrue(questType.isApplicableTo(node(
+            tags = mapOf(
+                "name" to "rundumdieuhr kiosk",
+                "opening_hours:signed" to "no",
+                "check_date:opening_hours" to "2021-03-01"
+            ),
+            timestamp = "2000-11-11".toCheckDate()?.toEpochMilli()
+        )))
+    }
```

Comment:
This test doesn't test what you want it to test. You want it to test that it takes the older of the two of `check_date:opening_hours` and `check_date:opening_hours:signed`, but to test this, `timestamp` would need to be "now" (i.e. the default). Otherwise the quest would anyway be applicable because the last edit date is 24 years old.

```suggestion
    @Test fun `is applicable if the opening hours not signed and only check date for OH exists`() {
        assertTrue(questType.isApplicableTo(node(
            tags = mapOf(
                "name" to "rundumdieuhr kiosk",
                "opening_hours:signed" to "no",
                "check_date:opening_hours" to "2021-03-01"
            )
        )))
    }
```